### PR TITLE
stix2 3.0.1 ❄️ 

### PR DESCRIPTION
**Pull request by psteyer**
stix2 3.0.1 ❄️ 

**Destination channel:** Snowflake

### Links

- [PKG-4121](https://anaconda.atlassian.net/browse/PKG-4121) 
- [Upstream repository](https://github.com/oasis-open/cti-python-stix2/)

### Explanation of changes:

- Initial package build
- added tests
- added ❄️ destination channel in abs

[PKG-4121]: https://anaconda.atlassian.net/browse/PKG-4121?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by SC426 — APPROVED** [review]
LGTM!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    - pip
+    - python
+    - wheel
+    - setuptools
+  run:
+    - python
+    - pytz
+    - requests
+    - simplejson
+    - stix2-patterns >=1.2.0
```
Comment (cbouss) [inline recipe/meta.yaml:29]:
```suggestion
    - stix2-patterns >=1.2.0
  run_constrained:
    - taxii2-client >=2.3.0
```

Comment (cbouss) [reply recipe/meta.yaml:29]: https://github.com/oasis-open/cti-python-stix2/blob/v3.0.1/setup.py#L63C20-L63C40

**Review by cbouss — APPROVED** [review]
LGTM, just a small suggestion to add run_constrained.

**Review by lorepirri — APPROVED** [review]
LGTM, besides Charles comment

**Pull request merged**